api/v1/customTypes/int10.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function (mongoose) {
+    function Int10(key, options) {
+        mongoose.SchemaType.call(this, key, options, 'Int10');
```

Comment:
I think it's really better to rename that "uint10" for the future users / developers to understand correctly that we can't have negative values